meta/saimetadatatypes.h

```diff
@@ -495,8 +515,62 @@ typedef enum _sai_attr_value_type_t
      * @brief Attribute value is the POE port consumption data.
      */
     SAI_ATTR_VALUE_TYPE_POE_PORT_POWER_CONSUMPTION,
+	
+	/**
+	* @brief Attribute value is spectrum power list.
+     */
+    SAI_ATTR_VALUE_TYPE_SPECTRUM_POWER_LIST,
 } sai_attr_value_type_t;
 
+/**
+ * @brief Defines statistics value unit.
+ */
+typedef enum _sai_stat_value_unit_t
+{
+    /**
+     * @brief Value unit is not dBm or dB.
+     */
+    SAI_STAT_VALUE_UNIT_NORMAL,
+
+    /**
+     * @brief Value unit is dBm.
+     */
+    SAI_STAT_VALUE_UNIT_DBM,
+
+    /**
+     * @brief Value unit is dB.
+     */
+    SAI_STAT_VALUE_UNIT_DB,
+
+} sai_stat_value_unit_t;
```

Comment:
this does not seem to be future expandable, why it's in metadata types instead os actual sai headers? this seem to not belong here